modules/ROOT/pages/monitoring/logging.adoc

```diff
@@ -277,30 +288,34 @@ See <<general-logging-appenders>> for more information.
 <4> `filePattern` specifies a file pattern to be used when the file is rolled.
 The pattern renames the files to _debug.log.01_ and _http.log.01_ when they reach the defined trigger.
 
-<5> `PatternLayout` defines the layout for the appender, in this case with the `GMT+2` timezone.
+<5> The `JsonTemplateLayout` layout is used to format the log messages in JSON format.
+Starting from Neo4j 2025.01, the default _debug.log_ format is JSON, and it is recommended to keep it enabled to ensure the best support.
+If you require the _debug.log_ messages in a different format, create an additional Appender.
```

Comment:
maybe here we should say that we recommend to create separate appended since default one is used by neo4j support.
Also maybe we should notice that the previous layout is there and commented out so to make migration back simpler if required?